jetty-ee9/jetty-ee9-annotations/src/main/java/module-info.java

```diff
@@ -17,7 +17,7 @@
     requires java.naming;
     requires org.slf4j;
     requires transitive  org.eclipse.jetty.annotations;
-    requires transitive  org.eclipse.jetty.ee;
+    requires transitive org.eclipse.jetty.ee.webapp;
```

Comment:
Weird spacing change.
Why are the others using 2 spaces?

Reply:
No idea. Fixing.